docusaurus/docs/quickstarts/zac/installation.md

```diff
@@ -33,32 +33,34 @@ you can perform the following steps.
     ```bash
     git clone https://github.com/openziti/ziti-console.git "${ZITI_HOME}/ziti-console"
     ```
-   
+
 2. Install node and npm and get the server ready:
 
     ```bash
     cd "${ZITI_HOME}/ziti-console"
     sudo apt install npm nodejs -y
     npm install
     ````
-   
-3. Use the ziti-controller certificates for the Ziti Console:
+
+3. You may need a newer version of Node if the app complains about `await`. In that case, you could use [Node Version Manager](https://github.com/nvm-sh/nvm#readme) (`nvm`) instead of `apt` to install Node.

```

Comment:
this should just be expressed in the Prerequisites section.

Reply:
moved that part up to pre-reqs